### PR TITLE
025deg jra55 iaf 

### PR DESCRIPTION
**Pull request by anton-seaice**
Contributes to #17 

This adds 0.25 degree IAF

### CODE REVIEW

File: metadata.yaml [hunk 1]
```diff
@@ -10,22 +10,26 @@ description: |-
...
-license: CC-BY-4.0
 keywords:
-  - quarter
-  - cosima
-  - access-om2-025
-  - jra55-do-v1.4.0
-  - iaf
+- JRA55
+- access-om2-025
+- repeat-year
```
Comment (anton-seaice) [inline metadata.yaml:25]:
```suggestion
- interannual
```

File: config.yaml [hunk 2]
```diff
@@ -37,41 +45,63 @@ submodels:
 
     - name: ocean
       model: mom
-      exe: /g/data/ik11/inputs/access-om2/bin/fms_ACCESS-OM_730f0bf_libaccessom2_d750b4b.x
-      input: /g/data/ik11/inputs/access-om2/input_20230515_025deg_topog/mom_025deg
+      exe: /g/data/vk83/apps/spack/0.20/release/linux-rocky8-x86_64/intel-19.0.5.281/mom5-git.2023.11.09=2023.11.09-ewcdbrfukblyjxpkhd3mfkj4yxfolal4/bin/fms_ACCESS-OM.x
```
Comment (aidanheerdegen) [inline config.yaml:48]:
```suggestion
      exe: /g/data/vk83/apps/spack/0.20/release/linux-rocky8-x86_64/intel-19.0.5.281/mom5-git.2023.11.09=2023.11.09-qji4nlmr6utrribaiyhewe4je6mifguz/bin/fms_ACCESS-OM.x
```
Missed this before.

File: config.yaml [hunk 3]
```diff
@@ -37,41 +45,63 @@ submodels:
...
+          - /g/data/vk83/experiments/inputs/access-om2/ocean/tides/global.025deg/2020.05.30/roughness_amp.nc
+          - /g/data/vk83/experiments/inputs/access-om2/ocean/tides/global.025deg/2020.05.30/roughness_cdbot.nc
+          - /g/data/vk83/experiments/inputs/access-om2/ocean/surface_salt_restoring/global.025deg/2020.05.30/salt_sfc_restore.nc
       ncpus: 1456
 
     - name: ice
       model: cice5
-      exe: /g/data/ik11/inputs/access-om2/bin/cice_auscom_1440x1080_480p_edcfa6f_libaccessom2_d750b4b.exe
-      input: /g/data/ik11/inputs/access-om2/input_20230515_025deg_topog/cice_025deg
+      exe: /g/data/vk83/apps/spack/0.20/release/linux-rocky8-x86_64/intel-19.0.5.281/cice5-git.2023.10.19=2023.10.19-rh3xfkrgajya3ghtliacuhlx3pgvrzqs/bin/cice_auscom_1440x1080_48x40_480p.exe
```
Comment (aidanheerdegen) [inline config.yaml:67]:
```suggestion
      exe: /g/data/vk83/apps/spack/0.20/release/linux-rocky8-x86_64/intel-19.0.5.281/cice5-git.2023.10.19=2023.10.19-v3zncpqjj2gyseudbwiudolcjq3k3leo/bin/cice_auscom_1440x1080_48x40_480p.exe
```

**Review by aidanheerdegen — CHANGES_REQUESTED** [review]
Missed a couple of the exe paths needing to be updated.

**Review by aidanheerdegen — APPROVED** [review]
LGTM. Thanks.

**Comment by anton-seaice** [comment]
We missed the manifests :). Shall i revert the merge or make a new PR or other ?

**Pull request merged**